### PR TITLE
chore: add ui-config endpoint

### PR DESCRIPTION
**Pull request by gastonfournier**
## About the changes
This will help us validate the version in terraform to make sure we use at least the minimum required version of Unleash

Comments should guide the reviewer. Everything under client is autogenerated

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -27,14 +27,14 @@ jobs:
         with:
           go-version: "1.20"
       - name: Login to ghcr
-        if: startsWith(matrix.image, 'ghcr.io/ivarconr')
+        if: contains(matrix.image, 'unleash-enterprise')
```
Comment (gastonfournier) [inline .github/workflows/build.yml:30]: More resilient check

File: operations.yaml [hunk 2]
```diff
@@ -33,5 +33,5 @@ inverseOperationIds:
   - getRoleById
   - updateRole
   - deleteRole
-  # - setRolesForUser
-  # - setRolesForGroup
\ No newline at end of file
+  
+  - getUiConfig
```
Comment (gastonfournier) [inline operations.yaml:37]: Add the new operation

File: test/api_admin_ui_test.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	apiClient := testClient()
+
+	t.Run("Test AdminUIAPIService GetUiConfig", func(t *testing.T) {
+
+		resp, httpRes, err := apiClient.AdminUIAPI.GetUiConfig(context.Background()).Execute()
+
+		require.Nil(t, err)
+		require.NotNil(t, resp)
+		assert.Equal(t, 200, httpRes.StatusCode)
+		assert.Regexp(t, regexp.MustCompile(`^\d+\.\d+\.\d+(-.+)?$`), resp.Version)
```
Comment (gastonfournier) [inline test/api_admin_ui_test.go:32]: Tests!

**Review by sighphyre — APPROVED** [review]
G

**Pull request merged**